src/sagemaker/xgboost/defaults.py

```diff
@@ -14,7 +14,9 @@
 from __future__ import absolute_import
 
 XGBOOST_NAME = "xgboost"
+XGBOOST_VERSION_0 = "0.90"
 XGBOOST_VERSION_1 = "0.90-1"
-XGBOOST_LATEST_VERSION = "0.90-2"
-XGBOOST_SUPPORTED_VERSIONS = [XGBOOST_VERSION_1, XGBOOST_LATEST_VERSION]
+XGBOOST_VERSION_2 = "0.90-2"
```

Comment:
Since we don't actually use this outside of `XGBOOST_SUPPORTED_VERSIONS`, I would prefer not having a constant for it and directly adding the string to `XGBOOST_SUPPORTED_VERSIONS`.